{"uuid":"c59f8197-d4a6-47f5-8963-a62f79145107","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["d0bea6ff-87ad-4b0c-b42e-8f8d1b8787df"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"281f33ba-993d-47d5-be0f-d66361aff0ba-attachment","type":"image/png"},{"name":"Myscreenshot","source":"a3833cf5-9f88-4083-9f10-910275c35830-attachment"}],"parameters":[],"start":1696883652464,"stop":1696883652532},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883652532,"stop":1696883652532},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883652533,"stop":1696883652533},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883652534,"stop":1696883652534}],"start":1696883652457,"stop":1696883652534}